tensorflow/lite/tools/evaluation/tasks/README.md

```diff
@@ -0,0 +1,21 @@
+# TFLite Model Task Evaluation
```

Comment:
Thanks for contributing this file. Recently, we have described the purpose of these tools and how they are used in our tf website (https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness). So, could you adapt this short accordingly? Many thanks!

Reply:
+1. Thanks for doing this @Harsh188 !

Reply:
Thanks for the review!